backend/src/test/resources/searxng/config/settings.yml

```
@@ -0,0 +1,10 @@
+use_default_settings: true
+
+server:
+  limiter: false
+
+search:
+  formats:
+    - html
+    - json
+    - csv
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)